index.html

```diff
@@ -0,0 +1,234 @@
+                      </p>
+                  </div>
+              </div>
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fas fa-chalkboard-teacher"></i>
+                  </div>
+                  <div class="main-card-title">
+                      <p>
+                          TS Exhibition
+                      </p>
+                  </div>
+                  <div class="main-card-description">
+                      <p>
+                          Showcase your work and present your tech startups in front of investors and industry experts.
+                      </p>
+                  </div>
+              </div>
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="far fa-comments"></i>
+                  </div>
+                  <div class="main-card-title">
+                      <p>
+                          Forum
+                      </p>
+                  </div>
+                  <div class="main-card-description">
+                      <p>
+                          Have the time to share your thoughts and opinions with experts and tech entrepenuer for each topic.
+                      </p>
+                  </div>
+              </div>
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fab fa-battle-net"></i>
+                  </div>
+                  <div class="main-card-title">
+                      <p>
+                          Workshop
+                      </p>
+                  </div>
+                  <div class="main-card-description">
+                      <p>
+                          Showcase your work and present your tech startups in front of investors and industry experts.
+                      </p>
+                  </div>
+              </div>
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fas fa-network-wired"></i>
+                  </div>
+                  <div class="main-card-title">
+                      <p>
+                          TS Ignite
+                      </p>
+                  </div>
+                  <div class="main-card-description">
+                      <p>
+                          Get opportunities to network with CC affiliates around the world, also after the summit.
+                      </p>
+                  </div>
+              </div>
+          </div>
+      </div>
+      <button type="button" class="main-btn">
+          Join CC Global Summit 2015
+      </button>
+      <a href="https://google.com" class="main-link-btn">See the all program</a>
+  </div>
+</main>
+<div class="container speakers">
+  <div class="speaker-title">
+      <p>Featured Speakers</p>
+  </div>
+  <div class="speaker-container-card">
```

Comment:
Please ensure that each of your `speaker cards` is well aligned as in the original design.
Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163295130-c5fe7787-0ef8-43bc-b5a7-a752a318897a.png) | ![image](https://user-images.githubusercontent.com/10332499/163295041-c868a542-19b5-48b0-9790-08193cc8ee1c.png)